### ISSUE TITLE
Re-consider rendering of amenity=car_sharing

### ISSUE THREAD

**Issue by imagico** [gravitystorm/openstreetmap-carto#4913]
In #1901 we removed rendering of `amenity=car_sharing` (https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcar_sharing) based on discussion in #1891.

I think it might be worth to re-consider that decision.  Our understanding of the role of this style as well as the role of OpenStreetMap has developed since then quite a bit.  As has the practical handling of car sharing in different parts of the world.  

Back then the main argument was the limited usefulness of the information to the map users because users of car sharing would be using the proprietary apps of the operator.  This is questionable from today's perspective because (a) OpenStreetMap is not unlikely to be a data source in specific car sharing apps, (b) car sharing users are likely not exclusively using proprietary tools of a single operator - but also might make use of more generic navigation tools for intermodal transportation or simply general purpose maps and navigation tools - like OSM-Carto and (c) care sharing stations are also a target for non-users of car sharing services - for example to meet someone who is using car sharing.

Apart from that the world is diverse and not everywhere operators of such services will maintain their own proprietary GIS databases with location information and their own proprietary apps (and even if they did - those might be worse in quality than OSM).

This does not mean i suggest we simply add this back.  But we should IMO revisit the subject and newly evaluate the possibility of rendering `amenity=car_sharing`.  Problems i see:

* It needs to be evaluated if the tag is used consistently in different parts of the world.  In particular critical is the delineation towards `amenity=car_rental` (https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcar_rental) and `amenity=car_pooling` (https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcar_pooling).
* For practical usefulness it would be essential to display the name of the network/club the car sharing station belongs to.  There seems to be competing use of the tags `operator`, `brand` and `network` for that.  This is likely to be a major obstacle, in particular if we want to avoid a disaster like #4629.

**Comment by RainerBielefeld** [comment]
I can't find any good reason to hide car_sharing.
I want to see a pictogrm there, because there are lots of other radsons why people need info that there is a car sharing place. 
Not only customers who want to find locations of their car sharing supplier.

**Comment by imagico** [comment]
The issue description indicates what needs to be done to bring this issue forward.  Everyone is welcome to contribute to that.

**Comment by jocelynetienne** [comment]
> I can't find any good reason to hide car_sharing. I want to see a pictogrm there, because there are lots of other radsons why people need info that there is a car sharing place. Not only customers who want to find locations of their car sharing supplier.

Indeed. One usage e.g. is surveying locations when planning to move in some area. At that point, what one wants is to know whether they'll be able to join a car sharing service and have a decent offer nearby.

**Comment by gosgtan** [comment]
I'm restarting the thread a bit. Carpooling areas and car sharing zones really need to finally be displayed, it's very popular here in France, and it just feels like important information is being omitted. Hundreds of them are built every year. Can we move forward with the issue? However, I am not competent to make the modifications, I hope someone will take it over.
